scripts/valhalla_build_elevation

```diff
@@ -235,22 +289,45 @@ if __name__ == "__main__":
     elif args.verbosity >= 2:
         LOGGER.setLevel(logging.DEBUG)
 
+    config = None
+    if args.config:
+        with open(args.config) as f:
+            config = json.load(f)
+
+    if args.outdir:
+        elevation_fp = args.outdir
+    elif config is not None:
+        elevation_fp = Path(config["additional_data"]["elevation"] or "elevation")
+    else:
+        LOGGER.critical("Either config or outdir is required.")
+        sys.exit(1)
+
     if args.from_geojson:
         tiles = get_tiles_with_geojson(args.from_geojson)
     elif args.from_bbox:
         tiles = get_tiles_with_bbox(args.from_bbox)
     elif args.from_tiles:
-        tiles = get_tiles_with_graph(Path(config["mjolnir"]["tile_dir"]))
+        if config is not None:
+            tiles = get_tiles_with_graph(Path(config["mjolnir"]["tile_dir"]))
+        else:
+            LOGGER.critical("--from-tiles requires a config to be specified.")
+            sys.exit(1)
```

Comment:
really nit:
```suggestion
        if config is None:
            LOGGER.critical("--from-tiles requires a config to be specified.")
            sys.exit(1)
        tiles = get_tiles_with_graph(Path(config["mjolnir"]["tile_dir"]))
```

mostly to save one line I guess :sweat_smile: feel free to disregard